cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,32 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('/#/login?_k=r8hegh');
```

Comment:
Hardcoded url in the code. It would be better to move it to a config or .env file

Reply:
Understood